sklearn/tests/test_common.py

```diff
@@ -222,7 +222,7 @@ def test_transformer_n_iter():
             else:
                 yield check_transformer_n_iter, name, estimator
 
-
+@ignore_warnings(category=DeprecationWarning)
```

Comment:
Not so confident about this one - the test_get_params_invariance(): is more general than RandomizedPCA.